src/shivani/Assignment_15/Bank.java

```diff
@@ -0,0 +1,148 @@
+package shivani.Assignment_15;
+
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI*/
+public class Bank {
+	int ibalance = 2000;
+	int hBalance = 2000;
+	int bBalance = 4000;
+	int sBalance = 3000;
+	int idCount, hdCount, bdCount, sdCount;
+	int icCount, hcCount, bcCount, scCount;
+
+	void banking(String bankName, String op, int amt) {
+		switch (op) {
+		case "debit":
+			switch (bankName) {
+			case "ICICI":
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.